index.d.ts

```diff
@@ -16,6 +16,14 @@ declare namespace unhandled {
 		*/
 		readonly showDialog?: boolean;
 
+	
+		/**
+	   Filter option that receives an array of regex.
```

Comment:
This just describes the name and the type, which are already clear. It should describe what it does and its use-cases.